dkg/pedersen/types/messages.go

```diff
@@ -447,16 +447,16 @@ func (req DecryptRequest) Serialize(ctx serde.Context) ([]byte, error) {
 // ReencryptRequest share sent to a node in order to
 // reencrypt a secret with the given public key
 type ReencryptRequest struct {
-	// U is the point from the write-request
-	U kyber.Point
-	// Xc is the public key of the reader
+	// K is the point from the writer
```

Comment:
`writer` might be unclear here.
Is this point the ciphertext or the randomness ?